l10n_it_fiscalcode/view/report_invoice_document.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+	<template id="print_invoice_inherit" inherit_id="account.report_invoice_document">
+		<xpath expr="//div[@name='invoice_address']" position="after">
```

Comment:
@joezsweet why not inside `invoice_address` div, along with other customer data? You would probably save space